<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>591</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>481</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="sofLabel">
        <property name="text">
         <string>sof:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="sofLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>481</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="elfLabel">
        <property name="text">
         <string>elf:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="elfLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QToolButton" name="sofButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="elfButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="sopcButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>100</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>481</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="sopcLabel">
        <property name="text">
         <string>sopc:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sopcLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="stdout">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>592</width>
      <height>262</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Fantasque Sans Mono</family>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>450</y>
      <width>200</width>
      <height>125</height>
     </rect>
    </property>
    <property name="title">
     <string>ELF</string>
    </property>
    <widget class="QPushButton" name="flash_nios">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Flash</string>
     </property>
    </widget>
    <widget class="QPushButton" name="program_nios">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Program</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>200</width>
      <height>125</height>
     </rect>
    </property>
    <property name="title">
     <string>SOF</string>
    </property>
    <widget class="QPushButton" name="program_fpga">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Program</string>
     </property>
    </widget>
    <widget class="QPushButton" name="flash_fpga">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>100</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Flash</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
